lib/public/views/Runs/ActiveColumns/runsActiveColumns.js

```diff
@@ -492,4 +492,11 @@ export const runsActiveColumns = {
         format: (_, run) => formatAliceCurrent(run.aliceDipolePolarity, run.aliceDipoleCurrent),
         profiles: ['runsPerLhcPeriod', 'runsPerDataPass', 'runsPerSimulationPass'],
     },
+    aliceL3AndDipoleCurrent: {
+        name: 'L3 / Dipole',
+        visible: false,
+        filter: ({ aliceL3AndDipoleCurrentFilter }) => selectionDropdown(aliceL3AndDipoleCurrentFilter, { selectorPrefix: 'l3-dipole-current' }),
+        filterTooltip: 'Combinations of L3 and dipole currents of physics runs rounded to kiloamperes',
```

Comment:
I think that the name is explicit enough. Were you asked for this tooltip?

Reply:
I would not say reason of having dropdown with a few options there is obvious. This brings glimpse of information why and what's that

Reply:
I disagree, but ok